src/Avalonia.Base/Utilities/StringTokenizer.cs

```diff
@@ -44,7 +44,7 @@ public StringTokenizer(string s, char separator = DefaultSeparatorChar, string?
             }
         }
 
-        public string? CurrentToken => _tokenIndex < 0 ? null : _s.Substring(_tokenIndex, _tokenLength);
+        public ReadOnlySpan<char> CurrentToken => _tokenIndex < 0 ? ReadOnlySpan<char>.Empty : _s.AsSpan().Slice(_tokenIndex, _tokenLength);
```

Comment:
StringTokenizer is part of public API, for no good reason.
Which unfortunately is the reason why we can't change this property type.
As an alternative, you should be able to add `ReadOnlySpan<char> CurrentTokenSpan` property and use it instead.
While keeping old property as a string.

Reply:
This class should be removed from public API in 12.0. But we are not there yet.

Reply:
```
##[error]API compatibility errors between 'baseline/netstandard2.0/Avalonia.Base.dll' (left) and 'target/netstandard2.0/Avalonia.Base.dll' (right):
##[error]CP0002: Member 'string? Avalonia.Utilities.StringTokenizer.CurrentToken.get' exists on baseline/netstandard2.0/Avalonia.Base.dll but not on target/netstandard2.0/Avalonia.Base.dll
##[error]API breaking changes found. If those are intentional, the APICompat suppression file can be updated by specifying the '--generate-suppression-file' parameter.
##[error]Target "ValidateApiDiff" has thrown an exception
```

Reply:
@maxkatz6 done. 
Changing `[MaybeNullWhen(false)]` in `TryReadString` also affects the API? Or doesn't matter what value is returned from the "false" `Try*` function?

Reply:
Changing attributes is not a breaking change.